packages/sit-onyx/src/composables/useClipping.ts

```diff
@@ -0,0 +1,92 @@
+import { useGlobalEventListener } from "@sit-onyx/headless";
+import { computed, nextTick, onMounted, ref, type Ref } from "vue";
+
+export function useClipping(
+  popoverRef: Ref<HTMLElement | null>,
+  popoverWrapperRef: Ref<HTMLElement | null>,
+  popoverPosition: Ref<string>,
+  isVisible: Ref<boolean>,
+  clipping: Ref<boolean>,
+) {
+  const clippingStyles = ref("");
```

Comment:
```suggestion
 const clippingStyles = ref<CSSProperties>({});
```

Instead of replying on string concatenation, I'd prefer to use an object type for this